StandardSocketsHttpHandler/Net/Http/SocketsHttpHandler/AuthenticationHelper.NtAuth.cs

```diff
@@ -3,7 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections.Generic;
-using System.Net;
+using System.Diagnostics;

```

Comment:
This change seems like a mistake, is it not?

Reply:
Yes, these are leftovers from backporting another feature, sorry for leaving them in the PR, now they are gone.